.github/workflows/ci.yml

```diff
@@ -114,17 +135,19 @@ jobs:
           java -version
           [[ $HYDRA_LICENSE == floating-key=* ]] && mkdir -p $HOME/.triplequote && echo "$HYDRA_LICENSE" > "$HOME/.triplequote/hydra.license" || echo "Hydra license file was not created"
```

Comment:
A random comment, I think we are no longer running any hydra tests and potentially we could remove it. We should probably talk with the triple quote people.

Reply:
Huh, yea I just looked. Honestly I don't know hardly anything about Hydra. However seeing that it's a paid project I have no issues ripping all the hydra stuff out. Is it something that is needed? Something that people have requested? Who uses hydra? I'll create a follow-up issue on this.

Reply:
Honestly I have no idea if there are people that both use hydra and Bloop or how much it was requested :thinking: